frontend/common/src/components/PageHeader/PageHeader.tsx

```diff
@@ -11,16 +11,16 @@ const PageHeader: React.FC<PageHeaderProps> = ({ icon, children, ...rest }) => {
   const Icon = icon || null;
 
   return (
-    <Heading
-      level="h1"
-      data-h2-display="b(flex)"
-      data-h2-align-items="b(center)"
-      data-h2-margin="b(top, none) b(bottom, m)"
-      data-h2-justify-content="b(start)"
-      {...rest}
-    >
-      {Icon && <Icon className="page-header__icon" />}
-      <span>{children}</span>
+    <Heading level="h1" data-h2-padding="base(x2, 0, x1, 0)" {...rest}>
+      {Icon && (
+        <Icon
+          className="page-header__icon"
```

Comment:
Does this class, `page-header__icon`, exist somewhere other than commented out code?

Reply:
- [ ] remove `page-header__icon` class from `frontend/common/src/components/PageHeader/PageHeader.tsx` (https://github.com/GCTC-NTGC/gc-digital-talent/pull/3051#discussion_r949357040)